projects/swimlane/ngx-ui/src/lib/components/json-editor/json-editor-flat/json-editor-node-flat/node-types/object-node-flat/object-node-flat.component.ts

```diff
@@ -247,6 +264,7 @@ export class ObjectNodeFlatComponent extends ObjectNode implements OnInit, OnCha
 
   private updateSchemaPropertyName(schema: JSONEditorSchema, newName: string, oldName: string): void {
     this.updateRequiredProperties(schema, newName, oldName);
+    schema.properties = (schema || {}).properties || {};
```

Comment:
Seams to me if `schema` is undefiend (hence `schema || {}`) then `schema.properties =` will fail.  I'd suggest the same as suggested above.

Reply:
Added the null check here as well. Also added it on the `updateRequiredProperties()` which had a similar check. Thanks!